x/ecocredit/simulation/genesis.go

```diff
@@ -52,7 +53,24 @@ func genAllowListEnabled(r *rand.Rand) bool {
 	return r.Int63n(101) <= 90
 }
 
-func genCreditTypes(r *rand.Rand) []*ecocredit.CreditType {
+func genCreditTypes(r *rand.Rand) []*core.CreditType {
+	return []*core.CreditType{
+		{
+			Name:         "carbon",
+			Abbreviation: "C",
+			Unit:         "metric ton CO2 equivalent",
+			Precision:    6,
+		},
+		{
+			Name:         "biodiversity",
+			Abbreviation: "BIO",
+			Unit:         "ton",
+			Precision:    6, // TODO: randomize precision, precision is currently locked to 6
+		},
+	}
+}
+
+func genCreditTypesLegacy(r *rand.Rand) []*ecocredit.CreditType {
```

Comment:
why do we still need this legacy function?

Reply:
We need to finish migrating genesis from v1alpha1 to v1 (see #977). @aleem1314 can you make sure we remove the use of legacy types in #977?